models/ed/R/met2model.ED2.R

```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```

Comment:
Ann, if you could change back that one line and push it to Github I'd be happy to pull the rest. Thank's for the update!